docs/product-assembly_pam-on-openshift/src/main/asciidoc/ba-openshift-overview-con.adoc

```diff
@@ -0,0 +1,35 @@
+[id='ba-openshift-overview-con']
+= Overview
+If you have an OpenShift environment, you can deploy {PRODUCT} into this environment.
+
+In this solution, components of {PRODUCT} are deployed as separate OpenShift pods. You can scale each of the pods up and down individually, providing as few or as many containers as necessary for a particular component. You can use standard OpenShift methods to manage the pods and balance the load.
+
+The following key components are available as pods running in OpenShift:
+
+* {KIE_SERVER}, also known as _KIE Server_, is the infrastructure element that runs a process or a group of processes. All process logic runs on execution servers.
```

Comment:
Would it be a good idea t say also known as _KIE Server_ or _execution server_? The term is used later without being part of the definition.

Reply:
{KIE_SERVER} translates to "Execution Server" when built for PAM, and to "Decision Server" when built for DM. Just a neat trick of our build system.

Reply:
Turns out it translates to Process Server. Need to discuss the Process Server and Execution Server terms.